docs/user-guide/ui.md

```diff
@@ -2,48 +2,82 @@
 ---
 
 
-Kubernetes has a web-based user interface that allows you to deploy containerized
-applications to a Kubernetes cluster, troubleshoot them, and manage the cluster itself.
+The web-based user interface of Kubernetes (Dashboard) allows you to deploy containerized applications to a Kubernetes cluster, troubleshoot them, and manage the cluster and its resources itself. You can use it for getting an overview of applications running on the cluster, as well as for creating or modifying individual Kubernetes resources and workloads, such as Daemon sets, Pet sets, Replica sets, Jobs, Replication controllers and corresponding Services, or Pods.
 
-By default, the Kubernetes Dashboard is deployed as a cluster addon. It is enabled by default in Kubernetes 1.2 clusters. Click [here](/docs/user-guide/ui-access/) to learn more about the Dashboard access.
+The Dashboard also provides information on the state of Pods, Replication controllers, etc. and on any errors that might have occurred. You can inspect and manage the Kubernetes resources, as well as your deployed containerized applications. You can also change the number of replicated Pods, delete Pods, and deploy new applications using a deploy wizard.

```

Comment:
If we are using 'Dashboard' as a proper name (without qualifying it for precision as the dashboard of Kubernetes), then we should probably drop the article 'the' -- 'Dashboard' rather than 'The Dashboard'. I think this is the format you used in the previous paragraph.